src/PKSim.Presentation/Presenters/Individuals/CreateIndividualPresenter.cs

```diff
@@ -92,9 +90,58 @@ public override void WizardNext(int previousIndex)
          base.WizardNext(previousIndex);
       }
 
-      public PKSim.Core.Model.Individual Individual
+      public Individual Individual
       {
          get { return PresenterAt(IndividualItems.Settings).Individual; }
       }
+
    }
-}
\ No newline at end of file
+
+   public class CreateIndividualPresenter : AbstractCreateIndividualPresenter
+   {
+      public CreateIndividualPresenter(ICreateIndividualView view, ISubPresenterItemManager<IIndividualItemPresenter> subPresenterItemManager,
+         IDialogCreator dialogCreator,
+         IBuildingBlockPropertiesMapper propertiesMapper, IObjectBaseDTOFactory buildingBlockDTOFactory)
+         : base(view, subPresenterItemManager, dialogCreator, propertiesMapper, buildingBlockDTOFactory, IndividualItems.All)
+      { }
+
+      protected override void UpdateControls(int indexThatWillHaveFocus)
+      {
+         UpdateViewStatus();
+         _view.NextEnabled = PresenterAt(IndividualItems.Settings).CanClose && indexThatWillHaveFocus != IndividualItems.Expression.Index;
+         _view.OkEnabled = CanClose;
+         _view.SetControlEnabled(IndividualItems.Expression, PresenterAt(IndividualItems.Settings).IndividualCreated);
+         _view.SetControlEnabled(IndividualItems.Parameters, PresenterAt(IndividualItems.Settings).IndividualCreated);
+      }
+   }
+   }
+
+//try moving to the Starter namespace
+   public class CreateIndividualPresenterForMoBi : AbstractCreateIndividualPresenter
+   {
+      public CreateIndividualPresenterForMoBi(ICreateIndividualView view, ISubPresenterItemManager<IIndividualItemPresenter> subPresenterItemManager,
+         IDialogCreator dialogCreator,
+         IBuildingBlockPropertiesMapper propertiesMapper, IObjectBaseDTOFactory buildingBlockDTOFactory)
+         : base(view, subPresenterItemManager, dialogCreator, propertiesMapper, buildingBlockDTOFactory, IndividualItems.AllExceptExpression)
+      {}
+      
+      protected override void UpdateControls(int indexThatWillHaveFocus)
+      {
+         UpdateViewStatus();
+         _view.NextEnabled = PresenterAt(IndividualItems.Settings).CanClose;
```

Comment:
the condition from above was removed. So next is enabled on the last tab, which does not make sense.